shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Tick.hs

```diff
@@ -144,44 +153,53 @@ validatingTickTransition nes slot = do
     ei <- asks epochInfo
     epochInfoEpoch ei slot
 
-  nes' <- trans @(Core.EraRule "NEWEPOCH" era) $ TRC ((), nes, epoch)
+  nes' <- trans @(Core.EraRule "NEWEPOCH" era) $ TRC (slot, nes, epoch)
```

Comment:
Duplicating comment about using `epoch` here instead of `slot`, which seems much more natural for a call to a "new epoch" rule!

Reply:
this has gone away.

Reply:
this has gone away.